Adding info for SDS through the Agent (#23600)
DataDog/documentation: content/en/sensitive_data_scanner/_index.md

Sensitive Data Scanner is a stream-based, pattern matching service used to identify, tag, and optionally redact or hash sensitive data. Security and compliance teams can implement Sensitive Data Scanner as a new line of defense, helping prevent against sensitive data leaks and limiting non-compliance risks.

To use Sensitive Data Scanner, set up a scanning group to define what data to scan and then set up scanning rules to determine what sensitive information to match within the data.

This document walks you through the following:

- The permissions required to view and set up Sensitive Data Scanner.
- Setting up scanning for sensitive data.
- Using the out-of-the-box dashboard.

**Note**: See [PCI DSS Compliance][1] for information on setting up a PCI-compliant Datadog organization.

{{< img src="sensitive_data_scanner/sds_main_12_01_24.png" alt="The Sensitive Data Scanner page showing six out of the 12 active scanning groups" style="width:90%;">}}

## Set up Sensitive Data Scanner
<br>
{{< callout url="https://google.com" >}}

>   Sensitive Data Scanner in the Agent is currently in private beta. To request access, full out this form.

Reviewer: ```suggestion
  Sensitive Data Scanner in the Agent is in private beta. To request access, full out this form.
```

{{< /callout >}} 

There are two locations where you can redact your sensitive data:

> - In the cloud:
>     - With **Sensitive Data Scanner in the Cloud**, you submit your logs in the Datadog backend. In this method, logs leave your premises before they are redacted. You can have multiple scanning groups per organization, and you can create custom scanning rules. You can also redact sensitive data in tags.
> - In your environment:
>     - With **Sensitive Data Scanner through the Agent**, Datadog redacts your logs before submitting them to the Datadog backend, and unredacted logs never need to leave your premises. With this method, you are limited to one scanning group per organization, and you can use only predefined library rules.

Reviewer: Optional formatting suggestion

```suggestion
**In the cloud**:
    - With **Sensitive Data Scanner in the Cloud**, you submit your logs in the Datadog backend. In this method, logs leave your premises before they are redacted. You can have multiple scanning groups per organization, and you can create custom scanning rules. You can also redact sensitive data in tags.

**In your environment**:
    - With **Sensitive Data Scanner through the Agent**, Datadog redacts your logs before submitting them to the Datadog backend, and unredacted logs never need to leave your premises. With this method, you are limited to one scanning group per organization, and you can use only predefined library rules.
```

    - Another way to redact your sensitive data in your environment before shipping to your downstream destinations is by using [Observability Pipelines][14].

### Prerequisites

{{< tabs >}}
{{% tab "In the Cloud" %}}
By default, users with the Datadog Admin role have access to view and set up scanning rules. To allow other users access, grant the `data_scanner_read` or `data_scanner_write` permissions under [Compliance][1] to a custom role. See [Access Control][2] for details on how to set up roles and permissions.

> {{< img src="sensitive_data_scanner/read_write_permissions.png" alt="The compliance permissions sections showing data scanner read and writer permissions" style="width:55%;">}}

Reviewer: Optional formatting, I like bigger images
```suggestion
{{< img src="sensitive_data_scanner/read_write_permissions.png" alt="The compliance permissions sections showing data scanner read and writer permissions" style="width:80%;">}}
```

[1]: /account_management/rbac/permissions/#compliance
[2]: /account_management/rbac/
{{% /tab %}}
{{% tab "In the Agent" %}}

1. Grant appropriate permissions. By default, users with the Datadog Admin role have access to view and set up scanning rules. To allow other users access, grant the `data_scanner_read` or `data_scanner_write` permissions under [Compliance][1] to a custom role. See [Access Control][2] for details on how to set up roles and permissions.

>     {{< img src="sensitive_data_scanner/read_write_permissions.png" alt="The compliance permissions sections showing data scanner read and writer permissions" style="width:55%;">}}

Reviewer: ```suggestion
    {{< img src="sensitive_data_scanner/read_write_permissions.png" alt="The compliance permissions sections showing data scanner read and writer permissions" style="width:80%;">}}
```

For Terraform, see the [Datadog Sensitive Data Scanner group][3] resource.

To set up a scanning group, perform the following steps:

1. Navigate to the [Sensitive Data Scanner][1] configuration page.
1. Click **Add scanning group**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Group**.
1. Enter a query filter for the data you want to scan. At the top, click **APM Spans** to preview the filtered spans. Click **Logs** to see the filtered logs.
1. Enter a name and description for the group.
1. Click the toggle buttons to enable Sensitive Data Scanner for the products you want (for example, logs, APM spans, RUM events, and Datadog events).
1. Click **Create**.

[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
[2]: /logs/explorer/search_syntax/
[3]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_group
{{% /tab %}}
{{% tab "In the Agent" %}}

> <div class="alert alert-warning"><strong>Note</strong>: Sensitive Data Scanner in the Agent supports only one scanning group per organization.</div>

Reviewer: Alert div implies the bolded Note
```suggestion
<div class="alert alert-warning">Sensitive Data Scanner in the Agent supports only one scanning group per organization.</div>
```

Author: I find that people often skip over callout boxes, and I think the bold text helps a little. Or so I like to believe. :)

1. Grant appropriate permissions. By default, users with the Datadog Admin role have access to view and set up scanning rules. To allow other users access, grant the `data_scanner_read` or `data_scanner_write` permissions under [Compliance][1] to a custom role. See [Access Control][2] for details on how to set up roles and permissions.

    {{< img src="sensitive_data_scanner/read_write_permissions.png" alt="The compliance permissions sections showing data scanner read and writer permissions" style="width:55%;">}}
2. Follow the steps to [enable remote configuration][3].
3. Install the Datadog Agent v7.54 or newer.

[1]: /account_management/rbac/permissions/#compliance
[2]: /account_management/rbac/
[3]: /agent/remote_config/?tab=configurationyamlfile#enabling-remote-configuration
{{% /tab %}}
{{< /tabs >}}

### Add a scanning group

{{< tabs >}}
{{% tab "In the Cloud" %}}
A scanning group determines what data to scan. It consists of a query filter and a set of toggles to enable scanning for logs, APM, RUM, and events. See the [Log Search Syntax][2] documentation to learn more about query filters.

For Terraform, see the [Datadog Sensitive Data Scanner group][3] resource.

> To set up a scanning group, perform the following steps.

Reviewer 2: Maybe a colon before the list:

```suggestion
To set up a scanning group, perform the following steps:
```

1. Enter a name and description for the group.
1. Click **Save**.

[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration/agent
[2]: /logs/explorer/search_syntax/
[3]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_group
{{% /tab %}}
{{< /tabs >}}

By default, a newly-created scanning group is disabled. To enable a scanning group, click the corresponding toggle on the right side.

### Add scanning rules

{{< tabs >}}
{{% tab "In the Cloud" %}}
A scanning rule determines what sensitive information to match within the data defined by a scanning group. You can add predefined scanning rules from Datadog's Scanning Rule Library or create your own rules using regex patterns. The data is scanned at ingestion time during processing. For logs, this means the scan is done before indexing and other routing decisions.

For Terraform, see the [Datadog Sensitive Data Scanner rule][2] resource.

> To add scanning rules, perform the following steps.

Reviewer 2: ```suggestion
To add scanning rules, perform the following steps:
```

Sensitive Data Scanner is a stream-based, pattern matching service used to identify, tag, and optionally redact or hash sensitive data. Security and compliance teams can implement Sensitive Data Scanner as a new line of defense, helping prevent against sensitive data leaks and limiting non-compliance risks.

To use Sensitive Data Scanner, set up a scanning group to define what data to scan and then set up scanning rules to determine what sensitive information to match within the data.

This document walks you through the following:

- The permissions required to view and set up Sensitive Data Scanner.
- Setting up scanning for sensitive data.
- Using the out-of-the-box dashboard.

**Note**: See [PCI DSS Compliance][1] for information on setting up a PCI-compliant Datadog organization.

{{< img src="sensitive_data_scanner/sds_main_12_01_24.png" alt="The Sensitive Data Scanner page showing six out of the 12 active scanning groups" style="width:90%;">}}

## Set up Sensitive Data Scanner
<br>
{{< callout url="https://google.com" >}}

>   Sensitive Data Scanner using the Agent is in private beta. To request access, full out this form.

Reviewer 2: ```suggestion
  Sensitive Data Scanner using the Agent is in private beta. To request access, fill out this form.
```

> {{< /callout >}} 

Reviewer 3: Could we move it between  `*In your environment:**` and `- With **Sensitive ` (line 56)? 
This way, it will be right before it is first mentioned

> {{< /callout >}} 

Author: Sure, that works

There are two locations where you can redact your sensitive data:

**In the cloud:**

- With **Sensitive Data Scanner in the Cloud**, you submit your logs in the Datadog backend. In this method, logs leave your premises before they are redacted. You can have multiple scanning groups per organization, and you can create custom scanning rules. You can also redact sensitive data in tags.

**In your environment:**

- With **Sensitive Data Scanner using the Agent**, Datadog redacts your logs before submitting them to the Datadog backend, and unredacted logs never need to leave your premises. With this method, you are limited to one scanning group per organization, and you can use only predefined library rules.

>    {{< img src="sensitive_data_scanner/sds_using_agent.png" alt="A diagram showing the flow of data from a customer environment to Datadog, where logs are redacted using the Agent before they leave the customer environment" style="width:80%;">}}

Reviewer 3: This image does not illustrate that the data is redacted in the environment with the agent. It mostly show what is the difference between On-prem vs In-Cloud. I wonder if this image can be put before the `In the cloud` section. 
Another option is to change the names in the arrow from `Metrics, Logs, Etc.` to `LOGS` & `REDACTED LOGS` similar to the [second image](https://dd.slack.com/archives/C05AFJECCKS/p1718908957892549?thread_ts=1718893542.977289&cid=C05AFJECCKS) (and ideally removing `unprocessed` word because `unprocessed` is not opposed to `redacted`)

Author: I cannot edit the words in the image. I can change it to the other image if you'd like, or move the image so another place.

Reviewer 3: Thanks for the context, I don't want to block the PR on this, I pinged @LutaoX  to see if this can be improved in a follow-up PR (or in this one if this update can be done quickly).

Reviewer 4: Agree with @vinckama 's comment. Indeed , in the blog post, we are utilizing the currently included image to explain "on-prem" vs "cloud" but have a separate image to illustrate the "redaction activity" between OP and Agent,  see this link: 
https://drive.google.com/file/d/1FJxXDPvnEOaQa1k4DKvDD8tMyBOVedg0/view?usp=sharing

Author: Removing image for now.

1. Click **Create**.

[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
[2]: /logs/explorer/search_syntax/
[3]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_group
{{% /tab %}}
{{% tab "Using the Agent" %}}
<div class="alert alert-warning"><strong>Note</strong>: Sensitive Data Scanner using the Agent supports only one scanning group per organization.</div>

A scanning group determines what logs to scan. It consists of a query filter to match eligible agents based on host tags.

To set up a scanning group, perform the following steps:

1. Navigate to the [Sensitive Data Scanner using the Agent][1] configuration page.
1. Click **Add scanning group**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Group**.
1. Enter a query filter for the data you want to scan. You can use only host-level tags for matching agents. At the bottom, the number of matching and eligible agents is displayed, including the total number out of all agents that match the tag.
1. Enter a name and description for the group.

> 1. Click **Save**.

Reviewer 3: It should be the same name as for cloud. I will see with the frontend if we can change the keyword here and have the same for Cloud and On-prem
```suggestion
1. Click **Create**.
```

Author: I don't want to change this until the actual app is changed.

Reviewer 3: sounds good to me, I open another PR once it's changed on the frontend  (most probably next week)

Author: Sounds good

{{% sds-scanning-rule %}}
1. Click **Add Rule**.

**Notes**:

- Any rules that you add or update affect only data coming into Datadog after the rule was defined.
- Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.
- After rules are added, ensure that the toggles for your scanning groups are enabled to begin scanning.

[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
[2]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_rule
{{< /collapse-content >}} 
{{% /tab %}}
{{% tab "In the Agent" %}}

A scanning rule determines what sensitive information to match within the data defined by a scanning group. You can add **up to 20** predefined scanning rules from Datadog's Scanning Rule Library. The Datadog Agent scans your data in your local environment during log collection, before logs are sent to the Datadog platform.

> <div class="alert alert-warning"><strong>Note</strong>: Sensitive Data Scanner in the Agent supports only predefined scanning rules from Datadog's Scanning Rule Library. The total number of scanning rules is limited to 20.</div>

Reviewer: ```suggestion
<div class="alert alert-warning">Sensitive Data Scanner in the Agent supports only predefined scanning rules from Datadog's Scanning Rule Library. The total number of scanning rules is limited to 20.</div>
```

{{% sds-scanning-rule %}}
1. Click **Add Rule**.

[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
[2]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_rule
{{< /collapse-content >}} 
{{% /tab %}}
{{% tab "In the Agent" %}}

A scanning rule determines what sensitive information to match within the data defined by a scanning group. You can add **up to 20** predefined scanning rules from Datadog's Scanning Rule Library. The Datadog Agent scans your data in your local environment during log collection, before logs are sent to the Datadog platform.

> For Terraform, see the [Datadog Sensitive Data Scanner rule][2] resource.

Reviewer 2: 🛑 SDS via Agent is not supported by TF, please remove.

To add scanning rules, perform the following steps.

> <div class="alert alert-warning"><strong>Note</strong>: Sensitive Data Scanner in the Agent supports only predefined scanning rules from Datadog's Scanning Rule Library. The total number of scanning rules is limited to 20.</div>

Reviewer 2: Can we move this up, before the "To add scanning rules, perform the following steps:"?

1. Optionally, add tags you want to associate with events where the values match the specified regex pattern. Datadog recommends using `sensitive_data` and `sensitive_data_category` tags. These tags can then be used in searches, dashboards, and monitors. See [Control access to logs with sensitive data](#control-access-to-logs-with-sensitive-data) for information on how to use tags to determine who can access logs containing sensitive information. **Note**: If no tags are added when saving the rule, the Agent automatically adds the `sds_agent` tag as well as `sensitive_data` and `sensitive_data_category` tags which are predefined by the rule.
1. Click **Save**.

[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration/agent
[2]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_rule
{{% /tab %}}
{{< /tabs >}}

**Notes**:

- Any rules that you add or update affect only data coming into Datadog after the rule was defined.
- Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.

> - After rules are added, ensure that the toggles for your scanning groups are enabled to begin scanning.

Reviewer 2: 🛑 "Notes" section shouldn't be shared; it should be in the tab:

> Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.

^ This is only true for Cloud SDS.

I'd suggest we keep original **Notes for the Cloud** tab:

> - Any rules that you add or update affect only data coming into Datadog after the rule was defined.
> - Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.
> - To turn off Sensitive Data Scanner entirely, set the toggle to **off** for each Scanning Group and Scanning Rule so that they are disabled.

And new **Notes for the Agent** tab:

> - Any rules that you add or update affect only data coming into Datadog after the rule was defined.
> - After rules are added, ensure that the toggles for your scanning groups are enabled to begin scanning.

Author: I still want to break disable out into its own step, which is true for both versions, so I think it's fine to remove that note from Cloud as well.

Author: And I think it's good to clarify that toggles should be enabled even in the Cloud version.

Reviewer 2: 👍 Agree.

{{% sds-scanning-rule %}}
1. Click **Add Rule**.

> **Notes**:

Reviewer 2: 🛑 Notes should be outside of "Add a custom scanning rule" collapsible section.

Author: Resolved

[2]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_rule
{{< /collapse-content >}} 

**Notes**:

- Any rules that you add or update affect only data coming into Datadog after the rule was defined.
- Sensitive Data Scanner does not affect any rules you define on the Datadog Agent directly.
- After rules are added, ensure that the toggles for your scanning groups are enabled to begin scanning.

{{% /tab %}}
{{% tab "Using the Agent" %}}

> A scanning rule determines what sensitive information to match within the data defined by a scanning group. You can add **up to 20** predefined scanning rules from Datadog's Scanning Rule Library. The Datadog Agent scans your data in your local environment during log collection, before logs are sent to the Datadog platform.

Reviewer 3: This information is exactly the same as the one described below. I suggested to remove it here. The section is more explicit:
 ```suggestion
A scanning rule determines what sensitive information to match within the data defined by a scanning group. The Datadog Agent scans your data in your local environment during log collection, before logs are sent to the Datadog platform.
```

1. Navigate to the [Sensitive Data Scanner using the Agent][1] configuration page.
1. Click **Add Scanning Rule**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Rule**.
1. In the **Add library rules to the scanning group** section, select the library rules you want to use. Use the **Filter library rules** input to search existing library rules. Next to the rule name you can find the list of predefined tags for each rule.
1. In the **Define rule target and action** section, select the action that you want to take for the matched sensitive information. **Note**: Redaction, partial redaction, and hashing are all irreversible actions.
    - **Redact**: Replaces all matching values with the text you specify in the **Replacement text** field.
    - **Partially Redact**: Replaces a specified portion of all matched data. In the **Redact** section, specify the number of characters you want to redact and which part of the matched data to redact.
    - **Hash**: Replaces all matched data with a unique identifier. The UTF-8 bytes of the match is hashed with the 64-bit fingerprint of FarmHash.

> 1. Optionally, add tags you want to associate with events where the values match the specified regex pattern. Datadog recommends using `sensitive_data` and `sensitive_data_category` tags. These tags can then be used in searches, dashboards, and monitors. See [Control access to logs with sensitive data](#control-access-to-logs-with-sensitive-data) for information on how to use tags to determine who can access logs containing sensitive information. **Note**: If no tags are added when saving the rule, the Agent automatically adds the `sds_agent` tag as well as `sensitive_data` and `sensitive_data_category` tags which are predefined by the rule.

Reviewer 5: ```suggestion
1. Optionally, add tags you want to associate with events where the values match the specified regex pattern. Datadog recommends using `sensitive_data` and `sensitive_data_category` tags. These tags can then be used in searches, dashboards, and monitors. See [Control access to logs with sensitive data](#control-access-to-logs-with-sensitive-data) for information on how to use tags to determine who can access logs containing sensitive information. **Note**: The Agent automatically adds the `sds_agent` tag as well as `sensitive_data` and `sensitive_data_category` tags which are predefined by the rule.
```

I believe the tags are added in addition to the default tags.

> 1. Optionally, add tags you want to associate with events where the values match the specified regex pattern. Datadog recommends using `sensitive_data` and `sensitive_data_category` tags. These tags can then be used in searches, dashboards, and monitors. See [Control access to logs with sensitive data](#control-access-to-logs-with-sensitive-data) for information on how to use tags to determine who can access logs containing sensitive information. **Note**: The Agent automatically adds the `sds_agent` tag as well as `sensitive_data` and `sensitive_data_category` tags which are predefined by the rule.

Reviewer 3: Let's remove any public mention to `sds_agent` as we will very likely deprecated this tag post private beta.
```suggestion
1. Optionally, add additional tags you want to associate with events where the values match the specified regex pattern. Datadog adds by default `sensitive_data` and `sensitive_data_category` tags which are predefined by the rule. These tags can then be used in searches, dashboards, and monitors. See [Control access to logs with sensitive data](#control-access-to-logs-with-sensitive-data) for information on how to use tags to determine who can access logs containing sensitive information.
```

1. Click **Save**.

**Notes**:

- Any rules that you add or update affect only data coming into Datadog after the rule was defined.
- After rules are added, ensure that the toggles for your scanning groups are enabled to begin scanning.

[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration/agent
[2]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_rule
{{% /tab %}}
{{< /tabs >}}

> See [Investigate Sensitive Data Issues][8] for details on how to use the [Summary][9] page to triage your sensitive data issues.

Reviewer 3: This section should only in the cloud part as the feature will not be accessible on the SDS in Agent for the private beta

### Edit scanning rules

{{< tabs >}}
{{% tab "In the Cloud" %}}

1. Navigate to the [Sensitive Data Scanner][1] configuration page.
1. Hover over the scanning rule you want to edit and click the **Edit** (pencil) icon.

   The **Define match conditions** section shows either the regular expression you wrote for your custom rule or an explanation of the library scanning rule you chose along with examples of matched sensitive information.
1. To make sure that a rule matches your data, you can provide a sample in the **Add sample data** section. If the rule finds matches in the sample data, a green **Match** label appears next to the input field.

> 1. Under **Create keyword dictionary**, you can add keywords to refine detection accuracy. For example, if you are scanning for a sixteen-digit Visa credit card number, you can add keywords like "visa", "credit", and "card". You can also require that these keywords appear within a specified number of characters before a match. By default, keywords must be within 30 characters before a match.

Reviewer 2: Use code formatting (eg. `visa`) for keywords, to be consistent with existing format (https://github.com/DataDog/documentation/blob/master/layouts/shortcodes/sds-scanning-rule.md?plain=1#L1)

```suggestion
1. Under **Create keyword dictionary**, you can add keywords to refine detection accuracy. For example, if you are scanning for a sixteen-digit Visa credit card number, you can add keywords like `visa`, `credit`, and `card`. You can also require that these keywords appear within a specified number of characters before a match. By default, keywords must be within 30 characters before a match.
```

1. Optionally, under **Define rule target and action**, edit the tags that you want to associate with events where the values match the rule. Datadog recommends using `sensitive_data` and `sensitive_data_category` tags, which can be used in searches, dashboards, and monitors. See [Control access to logs with sensitive data](#control-access-to-logs-with-sensitive-data) for information on how to use tags to determine who can access logs that contain sensitive data.
1. For **Set priority level**, choose a value based on your business needs.
1. Click **Update**.

[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
{{% /tab %}}
{{% tab "Using the Agent" %}}

1. Navigate to the [Sensitive Data Scanner using the Agent][1] configuration page.
1. Hover over the scanning rule you want to edit and click the **Edit** (pencil) icon.

   The **Define match conditions** section shows an explanation of the library scanning rule you chose along with examples of matched sensitive information.

> 1. Under **Create keyword dictionary**, you can add keywords to refine detection accuracy. For example, if you are scanning for a sixteen-digit Visa credit card number, you can add keywords like "visa", "credit", and "card". You can also require that these keywords appear within a specified number of characters before a match. By default, keywords must be within 30 characters before a match.

Reviewer 2: Use code formatting (eg. `visa`) for keywords, to be consistent with existing format (https://github.com/DataDog/documentation/blob/master/layouts/shortcodes/sds-scanning-rule.md?plain=1#L1)

```suggestion
1. Under **Create keyword dictionary**, you can add keywords to refine detection accuracy. For example, if you are scanning for a sixteen-digit Visa credit card number, you can add keywords like `visa`, `credit`, and `card`. You can also require that these keywords appear within a specified number of characters before a match. By default, keywords must be within 30 characters before a match.
```

7. Optionally, add tags.
8. Click **Add Rules**.

[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
[2]: /logs/log_configuration/processors/?tab=ui#remapper
[3]: https://app.datadoghq.com/logs/pipelines
{{% /tab %}}
{{% tab "Using the Agent" %}}
This functionality is not available for Sensitive Data Scanner using the Agent.
{{% /tab %}}
{{< /tabs >}}

## Out-of-the-box dashboard

> When Sensitive Data Scanner is enabled, an [out-of-the-box dashboard][13] summarizing sensitive data findings is automatically installed in your account. To access this dashboard, go to **Dashboards > Dashboards List** and search for "Sensitive Data Scanner Overview".

Reviewer 3: For another PR, this section should be changed to a link to `Investigate Sensitive Data Issues`. The dashboard should be part of this doc. We should heavily recommend users to use the Summary page and heads-up the dashboard only for private beta users of SDS in Agent.

Reviewer 3: cc @PronoyC  if you can take a look to this part.

Author: I'm confused by this comment. Is there a requested change on my part?

Reviewer 3: Not on your part necessarily and definitely not in this PR. I've open a comment so we (Pronoy or I) can follow-up on this later.

Author: Acknowledged, thanks for clarifying

Reviewer 6: yeah I can resolve this comment -- it's something to update in a future PR instead
